zerver/webhooks/jotform/doc.md

```diff
@@ -1,19 +1,38 @@
+# Zulip Jotform integration
+
 Get Zulip notifications for your Jotform responses!
 
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
-   By default, the integration will use the form's title as the topic.
+1. {!generate-webhook-url-basic.md!}
 
 1. In Jotform, go to **Form Builder**, and click on **Settings** tab.
 
 1. Click on **INTEGRATIONS**, search for **WebHooks** and select it.
 
-1. Add the above constructed **url** to the field, and click the
+1. Add the **url** generated above to the field, and click the
    **COMPLETE INTEGRATION** button.
 
+!!! warn ""
```

Comment:
I think this can be placed near the original location instead, since the information is relevant to the URL generation.
Just indent it to make it appear as part of the step.
Also, we could use a **tip** instead of a **warning**?